L117-cpp-replacing-gpr-logging-with-abseil-logging.md

```diff
@@ -0,0 +1,94 @@
+L117-Replacing gpr logging with Abseil logging
+----
+
+* Author(s): tjagtap@google.com
+* Approver: roth@google.com , ctiller@google.com
+* Status: In Review 
+* Implemented in: gRPC C++
+* Last updated: April 29, 2024.
+* Discussion at: (filled after thread exists)
+
+## Abstract
+
+Replacing gpr logging and asserts with Abseil logging and asserts
+
+## Background & Rationale
```

Comment:
"Background" and "Rationale" should be two separate sections, as per the gRFC template.  The "Rationale" section is toward the bottom.

I think the rationale section should discuss the problem of initializing absl logging that was raised by @eugeneo.  The application will need to call `absl::InitializeLog()` itself, or else there will be a log message indicating that logging is happening before absl logging is initialized.  If in the future absl fixes https://github.com/abseil/abseil-cpp/issues/1656, then we could consider initializing absl logging ourselves.

Reply:
Fixed